api/src/opentrons/hardware_control/adapters.py

```diff
@@ -46,15 +48,23 @@ def __init__(self, asynchronous_instance: WrappedObj) -> None:
         """
         self._obj_to_adapt = asynchronous_instance
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return "<SynchronousAdapter>"
 
     @staticmethod
-    def call_coroutine_sync(loop, to_call, *args, **kwargs):
-        fut = asyncio.run_coroutine_threadsafe(to_call(*args, **kwargs), loop)
+    def call_coroutine_sync(
+        loop: asyncio.AbstractEventLoop,
+        to_call: WrappedFunc,
+        *args: Sequence[Any],
+        **kwargs: Mapping[str, Any]
```

Comment:
[You don't need to specify](https://mypy.readthedocs.io/en/stable/getting_started.html?highlight=kwargs#more-function-signatures) `Sequence` and `Mapping` when you're using `*args` and `**kwargs`

```suggestion
        *args: Any,
        **kwargs: Any,
```

Reply:
And in fact it breaks stuff, i wonder why this one didn't cause errors after I fixed the rest